awsutil/README.md

```diff
@@ -0,0 +1,37 @@
+# AWSUTIL - Go library for generating aws credentials
+
+*NOTE*: This is version 2 of the library. The `v0` branch contains version 0,
+which may be needed for legacy applications or while transitioning to version 2.
+
+## Usage
+
+Following is an example usage of generating AWS credentials with static user credentials
+
+```go
+
+// AWS access keys for an IAM user can be used as your AWS credentials.
+// This is an example of an access key and secret key
+var accessKey = "AKIAIOSFODNN7EXAMPLE"
+var secretKey = "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+
+// Access key IDs beginning with AKIA are long-term access keys. A long-term
+// access key should be supplied when generating static credentials.
+config, err := awsutil.NewCredentialsConfig(
+    awsutil.WithAccessKey(accessKey),
+    awsutil.WithSecretKey(secretKey),
+)
+if err != nil {
+    return err
+}
+
+s3Client := s3.NewFromConfig(config)
+
+```
+
+## Contributing to v0
+
+To push a bug fix or feature for awsutil `v0`, branch out from the `awsutil/v0` branch.
```

Comment:
Mind adding a link to the branch? Can go around the `awsutil/v0` text. Doesn't look like the branch exists yet.

Reply:
I created the branch and added a link to this line.